### DISCUSSION TITLE
Drive Sounds ON/OFF Toggle

### DISCUSSION THREAD

**Discussion by aZtOcKdOg** [vAmigaWeb/vAmigaWeb#241] [Ideas]
Would it be possible to add a toggle for drive sounds (hdd/fdd etc)? So that we can optionally have a quiet productivity setup for example...

It could live in settings just near drive speed

**Comment by mithrendal** [comment]
Or maybe drive Sound Volume setting ?

0%, 25%, 50%, 75%, 100%

On the other hand a toggle is even more simple.

**Comment by aZtOcKdOg** [comment]
Wow a vol setting for DRIVES (and PAULA #161) would be great too and benefit when balancing machine sounds with game / bg music!

But a toggle would suffice I think!

I wonder when DRIVE SOUNDS toggle = OFF (or VOL = 0) does the implementation of the drive sound stop too?
(Will that free up some iPad resources / CPU cycles etc? Same thoughts for Vol = 0 if a Paula vol slider was implemented)

**Comment by dirkwhoffmann** [comment]
The core emulator already supports the following options. It's just a matter of exposing these to the user via a nice GUI interface:

```
    OPT_AUD_PAN0,
    OPT_AUD_PAN1,
    OPT_AUD_PAN2,
    OPT_AUD_PAN3,
    OPT_AUD_VOL0,
    OPT_AUD_VOL1,
    OPT_AUD_VOL2,
    OPT_AUD_VOL3,
    OPT_AUD_VOLL,
    OPT_AUD_VOLR,

    OPT_DRIVE_PAN,
    OPT_DRIVE_STEP_VOLUME,
    OPT_DRIVE_POLL_VOLUME,
    OPT_DRIVE_INSERT_VOLUME,
    OPT_DRIVE_EJECT_VOLUME
```

Setting the drive volumes to 0 does *not* affect drive emulation, nor does setting any of the AUD volumes to zero affect Paula's emulation.
